cms/templates/js/self-paced-grading-editor.underscore

```diff
@@ -0,0 +1,43 @@
+<h3 class="modal-section-title"><%- gettext('Grading') %></h3>
```

Comment:
Going to wait to comment on this until we address my comment in the JS file regarding if these need to be combined.